### PR TITLE
scripts/drtprod: add drt-chaos create command

### PR DESCRIPTION
**Pull request by itsbilal**
This change adds the create command for the drt-chaos cluster to the drtprod script.

Epic: none

Release note: None

### CODE REVIEW

**Review by dt — APPROVED** [review]

File: scripts/drtprod [hunk 1]
```diff
@@ -86,6 +86,19 @@ case $1 in
...
         $0 dns drt-main create
         ;;
+      "drt-chaos")
+        roachprod create drt-chaos \
+          --clouds gce \
+          --gce-managed \
+          --gce-zones "us-east1-b" \
+          --nodes 5 \
+          --gce-machine-type n2-standard-8 \
+          --gce-pd-volume-size 3000 --local-ssd=false \
```
Comment (dt) [inline scripts/drtprod:96]: PD and not local ssd?

Comment (itsbilal) [reply scripts/drtprod:96]: We could do local ssd to save $ etc, but I feel like pd has been more reliable for us. Good with either though.

Comment (itsbilal) [reply scripts/drtprod:96]: Let's stick with pd due to the ongoing issues.

**Comment by itsbilal** [comment]
bors r=dt

**Pull request merged**